apps/web/src/styles/account.css

```
@@ -0,0 +1,1053 @@
+}
+
+.account-btn.btn-loading {
+  opacity: 0.7;
+  pointer-events: none;
+}
+
+.btn-loading-spinner {
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  border: 2px solid currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)